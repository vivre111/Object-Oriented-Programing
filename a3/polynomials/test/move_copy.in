print a
print b

create a -9/2 8 4/3 4 2/1 2 0/5 1
create b 3/5 0
create c 0/2 12
create d 9/-2 12
print a
print b
move_copy a b
print b
move_copy d c
print c
move_copy c d
print d
