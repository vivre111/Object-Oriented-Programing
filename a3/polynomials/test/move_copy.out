0
0
(-9/2)x^8 + (4/3)x^4 + (2)x^2
(3/5)
(-9/2)x^8 + (4/3)x^4 + (2)x^2
(-9/2)x^8 + (4/3)x^4 + (2)x^2
(-9/2)x^12
(-9/2)x^12
(-9/2)x^12
(-9/2)x^12
